### DISCUSSION TITLE
Coordinates of hyp3 interferogram

### DISCUSSION THREAD

**Discussion by DD-ZHAO** [insarlab/MintPy#1295] [Q&A] [answered]
Hello everyone. I used a Python script to crop the interferogram of hyp3 to the most common size, and the resulting corner coordinates are (253960.0, 4473880.0) in the upperleft corner and (545240.0, 4237640.0) in the lowerright corner. The numbers in these two sets of coordinates are neither latitude or longitude nor pixel numbers. What is their meaning? Thank you for your help.

**Comment by jhkennedy** [comment, marked as answer]
HyP3 products are distributed as UTM-projected GeoTIFs, so those coordinates are in meters. This should be stated in the product README, and a gdalinfo call will also tell you what UTM zone the products are in. For more info, see:
https://hyp3-docs.asf.alaska.edu/products/

**Reply by DD-ZHAO** [reply]
@jhkennedy Got it. Thank you very much for your reply!
